### ISSUE TITLE
ACC-264.1 (Level A) _Labels_ Associated texts are not read out. 

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3238]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Labels
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1 (Level A) _Labels_ Associated texts are not read out. 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Go to KubeconfigURL:https://kyma-env-broker.cp.stage.kyma.cloud.sap/kubeconfig/DFEFE4E0-E202-4455-8A61-37511B4D7553
•	Click on  the link https://dashboard.stage.kyma.cloud.sap/clusters and select Connect.
•	Upload the kubeconfing file downloaded in the first step. You can either drag and drop it or select the upload icon to choose it from the device.
•	Next Step
•	

3. Issue Type: Manual
Observed Behavior:
Associated texts are not read out on accessing the required field such as ‘Token’ and Switch button such ’OIDC provider’ under the wizard step ‘Authentication’ within the dialog ‘Connect Cluster’.
Screenshot:
 
Screenshot 2:
 
Expected Behavior:
The corresponding label and group label should be read out, when each & every UI is accessed.

Relationships and information conveyed by the presentation (such as labels) should be identified by assistive technologies so user can connect the accessed UI with associated label.
[ACC-264.1 (Level A) _Labels_ Associated texts are not read out.docx](https://github.com/user-attachments/files/16687515/ACC-264.1.Level.A._Labels_.Associated.texts.are.not.read.out.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16687516/Issue.processing-09.2021.2.docx)
